### PR TITLE
cron-expression-examples

### PR DESCRIPTION
**Pull request by kghamilton89**
## `cron` Expression Examples

* `cron` expression examples from [Scheduling](https://axibase.com/docs/axibase-collector/scheduling.html#cron-expression-examples) copied to ATSD Docs. 
* I also copied Field Control Symbols and Field Constraints [tables](https://axibase.com/docs/axibase-collector/scheduling.html#cron-expressions) because they are relevant.

### CODE REVIEW

File: shared/scheduling.md [hunk 1]
```diff
@@ -24,6 +24,35 @@ Either `0` or `7` can be used for Sunday in the `day-of-week` field.
...
+| `minute` | `0-59`, `R` |
+| `hour` | `0-23`, `R` |
+| `day-of-month` | `1-31`, `?` |
+| `month` | `1-12` or `JAN-DEC` |
+| `day-of-week` | `1-7` or `MON-SUN`, `?`  |
+
+* If a value is set in `day-of-week`, `day-of-month` must be set to `?`. For example `0 0 6 ? * MON`.
+* If a value is set in `day-of-month`, `day-of-week` must be set to `?`. For example `0 0 6 */2 * ?`.
+
+Second, minute, and hour fields support **R** (random) symbol to randomize execution time.
```
Comment (rodionos) [inline shared/scheduling.md]: **R** field is not supported in ATSD. It's a custom feature supported only in collector.

Comment (kghamilton89) [reply shared/scheduling.md]: Ah sorry. All **R** character information removed by `232e729`.

File: shared/scheduling.md [hunk 2]
```diff
@@ -32,10 +58,17 @@ The time zone is displayed on the **Settings > System Information** page.
...
+| `0/10 * * * * ?` | `0/10` | `*` | `*` | `*` | `*` | `?` | Every ten seconds. |
+| `0 0/1 * * * ?`  | `0` | `0/1` | `*` | `*` | `*` | `?` | Every minute. |
+| `0 0 0 * * ?`    | `0` | `0` | `0` | `*` | `*` | `?` | Every day at 00:00. |
+| `0 5,35 * * * ?` | `0` | `5,35` | `*` | `*` | `*` | `?` | Every hour at the fifth and thirty-fifth minute. |
+| `0 0 6 ? * MON`  | `0` | `0` | `6` | `?` | `*` | `MON` | Every Monday at 06:00. |
+| `0 5 0 * 8 ?`    | `0` | `5` | `0` | `*` | `8` | `?` | At 00:05 every day in August. |
+| `30 15 14 1 * ?` | `30`| `15`| `14`| `1` | `*` | `?` | At 14:15:30 on the first of every month. |
+| `0 0 22 ? * 1-5` | `0` | `0` | `22`| `?` | `*` | `1-5` | At 22:00 on Mon, Tue, Wed, Thu and Fri. |
+| `0 5 0-10/2 * * ?` | `0` | `5` | `0-10/2` | `*` | `*` | `?` | At every ninth minute past the zero, second, fourth, sixth, eighth, and 10th hour. |
+| `0 0 0,12 1 */2 ?` | `0` | `0` | `0,12`| `1` | `*/2` | `?` | At 00:00 and 12:00 on the first in <br/>January, March, May, July, September and November. |
```
Comment (rodionos) [inline shared/scheduling.md]: Is `*/2` for month equal to 1,3,5, or perhaps 2,4,6,...?

Comment (kghamilton89) [reply shared/scheduling.md]: According to the Description, it is equal to 1,3,5, etc. (January, March, May, etc.).

**Pull request merged**